pkg/dmn/dmn_engine_test.go

```diff
@@ -0,0 +1,111 @@
+package dmn_test
+
+import (
+	"errors"
+	"github.com/pbinitiative/zenbpm/pkg/dmn"
+	"gopkg.in/yaml.v3"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+type testCase struct {
+	Decision       string                 `yaml:"decision"`
+	Input          map[string]interface{} `yaml:"input"`
+	ExpectedOutput interface{}            `yaml:"expectedOutput"`
+}
+
+type configuration struct {
+	DMN   string
+	Tests []testCase `yaml:"tests"`
+}
+
+const BulkEvaluationTestPath = "./test-data/bulk-evaluation-test"
+
+func TestBulkEvaluation(t *testing.T) {
```

Comment:
Preferably rename to show what method is under test.
Because there will be more tests added for other functions in the interface later.

```suggestion
func Test_EvaluateDRD(t *testing.T) {
```

Reply:
@nitram509 I renamed it to Test_BulkEvaluateDRD as its a bulk test. Check if thats okay